README.md

```diff
@@ -50,104 +42,35 @@ python -m explainaboard_client.cli.evaluate_system \
-that are not supported by DataLab yet:
-
-```
-python -m explainaboard_client.cli.evaluate_system \
-  --username $EB_USERNAME --api_key $EB_API_KEY \
-  --task [TASK_ID] \
-  --system_name [MODEL_NAME] \
-  --system_output_file [SYSTEM_OUTPUT] --system_output_file_type [FILE_TYPE] \
-  --custom_dataset_file [CUSTOM_DATASET] --custom_dataset_file_type [FILE_TYPE] \
-  --source_language [SOURCE] --target_language [TARGET]
-```
-
-with similar file and file-type arguments to the system output above. If you're
-interested in getting your datasets directly supported within ExplainaBoard, please
-open an issue or send a PR to [DataLab](https://github.com/expressai/datalab), and we'll
-be happy to help out!
-
-**Finding Uploaded Systems:** You can also find systems that have already been evaluated
-using the following syntax
-```
-python -m explainaboard_client.cli.find_systems \
-  --username $EB_USERNAME --api_key $EB_API_KEY --output_format tsv
-```
-By default this outputs in a summarized TSV format (similar to the online system
-browser), but you can set `--output_format json` to get more extensive information.
-There are many options for how you can specify which systems you want to find, which you
-can take a look at by running `python -m explainaboard_client.cli.find_systems` without
-any arguments.
-
-**Deleting System Outputs:** You can delet existing system outputs using the following
-command:
-```
-python -m explainaboard_client.cli.delete_systems \
-  --username $EB_USERNAME --api_key $EB_API_KEY --system_ids XXX YYY
-```
-Here the `system_ids` are the unique identifier of each system returned in the
-`system_id` field of the JSON returned by the `find_systems` command above. The system
-IDs are *not* the system name as displayed in the interface.
-
-### Evaluating Systems on Benchmarks from the Command Line
-Instead of simply evaluating an individual system, another common scenario is 
-to submit a group of systems to a benchmark (e.g., GLUE). To achieve this goal,
-you can follow the command below: 
-
-```shell
-python -m explainaboard_client.cli.evaluate_benchmark \
-      --username XXX  \
-      --api_key YYY \
-      --system_name your_system \
-      --system_outputs submissions/* \
-      --benchmark benchmark_config.json \
-      --server local
+For more details on precisely how to specify all the variables, as well as how to do
+other things such as search for and delete systems from the command line, click through
+to the [command line documentation](docs/cli.md).
+
+## Programmatic Usage
+
+Below is an example of how you can evaluate a system using ExplainaBoard through your
+programs.
+
+```python
+import os
+import explainaboard_client
+
+# Set up your environment
+explainaboard_client.username = os.environ['EB_USERNAME']
+explainaboard_client.api_key = os.environ['EB_API_KEY']
+client = explainaboard_client.ExplainaboardClient()
+
+# Do the evaluation
```

Comment:
As a demonstration example, I think this would make the product more attractive: `shortest code makes most powerful things. Based on this excerpt code could be further simplified in the future, for example,
* We can reduce the need for `task` name, `source_language`, which could be obtained in named datasets
* We can even remove the `system_name`, `split`, `system_output_file_type`, by providing some default values.
Of course, we can instruct users to see more comprehensive usage of general examples.

Reply:
Opened an issue but will not fix in this PR. https://github.com/neulab/explainaboard_client/issues/47